core/sdk-core/src/main/java/software/amazon/awssdk/core/client/config/ClientOverrideConfiguration.java

```diff
@@ -141,6 +145,14 @@ public List<ExecutionInterceptor> executionInterceptors() {
         return executionInterceptors;
     }
 
+    /**
+     * The optional scheduled executor service that should be used for scheduling tasks such as async retry attempts
+     * and timeout task.
```

Comment:
Can we add javadocs to remind users that the scheduled executor service will not be closed when the client is closed, and that they are responsible for closing it?